crates/evm/traces/src/decoder/mod.rs

```diff
@@ -291,6 +291,31 @@ impl CallTraceDecoder {
         }
     }
 
+    /// Populates the traces with decoded data by mutating the
+    /// [CallTrace] in place. See [CallTraceDecoder::decode_function] and
+    /// [CallTraceDecoder::decode_event] for more details.
+    pub async fn populate_traces(&self, traces: &mut Vec<CallTraceNode>) {
+        for node in traces {
+            let decoded = self.decode_function(&node.trace).await;
+            node.trace.decoded.label = decoded.label;
+            node.trace.decoded.call_data = decoded.func;
+            node.trace.decoded.return_data = decoded.return_data;
```

Comment:
Can we unify the types here? Both function and event below, I'm OK with the methods returning the type and then setting it here

Reply:
Updated to have a unified `DecodedItem` type

Reply:
Sorry that is not what I meant. I meant we should just reuse the same types from revm-inspectors

Reply:
Ah yeah that makes a lot more sense lol